### PR TITLE
Css modules

### PR DESCRIPTION
**Pull request by FranckWebPro**

### CODE REVIEW

**Review by sorrento974 — APPROVED** [review]

**Comment by sorrento974** [comment]
ok

File: client/src/components/OrangeLink.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import styles from "../styles/OrangeLink.module.css";
+
+function OrangeLink({ name, path }) {
+  return (
+    <Link
+      to={path}
+      className={
+        path.includes("season") ? "orange-button" : styles.orangeButton
```
Comment (Yoann-Fortin) [inline client/src/components/OrangeLink.jsx:10]: Condition isn't necessary here

File: client/src/components/SectionAllCocktails.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+function SectionAllCocktails() {
+  return (
+    <section className="allCocktailsSection">
+      <img
+        src="./src/assets/images/allcocktails.svg"
+        alt="A plate with many cocktails of different colors"
+      />
+      <h2>All of Cocktail Club</h2>
+      <p>erbtrngyfgvshtrgvj,ngfnbfdhnf,ghfgvghdg,fgnhfa</p>
+      <div className="buttonBorder">
```
Comment (Yoann-Fortin) [inline client/src/components/SectionAllCocktails.jsx:12]: Tag for style isn't good

File: client/src/main.jsx [hunk 3]
```diff
@@ -1,21 +1,37 @@
...
+        element: <CocktailPage />,
+      },
+    ],
   },
 ]);
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
-
-root.render(
-  <React.StrictMode>
```
Comment (Yoann-Fortin) [inline client/src/main.jsx:18]: Is INTERDIT to delete this

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**